Dockerfile

```diff
@@ -22,10 +13,18 @@ ENV VARNISH_VARNISHD_PARAMS ''
 ENV VARNISH_VARNISHNCSA_PARAMS ''
 ENV VARNISH_CACHE_TAGS_HEADER Cache-Tags
 
+COPY patches /tmp/patches/
+COPY config /tmp/config/
+COPY scripts /tmp/scripts/
+COPY healthcheck.sh /usr/bin/healthcheck.sh
+COPY startup.sh /usr/bin/startup.sh
+
+RUN set -ex && apk add --update --no-cache bash && /tmp/scripts/build
```

Comment:
Please use the same RUN format as in other images, e.g.
https://github.com/docksal/service-vhost-proxy/blob/develop/Dockerfile#L45